osgp/platform/osgp-secret-management/src/main/java/org/opensmartgridplatform/secretmgmt/application/config/ApplicationConfig.java

```diff
@@ -0,0 +1,14 @@
+package org.opensmartgridplatform.secretmgmt.application.config;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+@Configuration
+@PropertySource("classpath:secret-management.properties")
+@PropertySource(value = "file:${osgp/Global/config}", ignoreResourceNotFound = true)
+@PropertySource(value = "file:${osgp/SecretManagement/config}", ignoreResourceNotFound = true)
```

Comment:
These @PropertySource annotations must be placed on @SpringBootApplication annotated class due to loading order problems